WcaOnRails/db/structure.sql

```diff
@@ -1,8 +1,8 @@
--- MySQL dump 10.16  Distrib 10.1.9-MariaDB, for Linux (x86_64)
+-- MySQL dump 10.13  Distrib 5.5.46, for debian-linux-gnu (x86_64)
```

Comment:
I copied your version of the file and replaced mine.
Is that the correct way? I didn't find the proper way to "remove" a file
from a commit.

On Mon, Jan 4, 2016 at 11:50 PM, Jeremy Fleischman <notifications@github.com
> wrote:

> In WcaOnRails/db/structure.sql
> <https://github.com/cubing/worldcubeassociation.org/pull/298#discussion_r48805334>
> :
>
> > @@ -1,8 +1,8 @@
> > --- MySQL dump 10.16  Distrib 10.1.9-MariaDB, for Linux (x86_64)
> > +-- MySQL dump 10.13  Distrib 5.5.46, for debian-linux-gnu (x86_64)
>
> It looks like nothing actually changed here, please remove this file from
> the commit.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cubing/worldcubeassociation.org/pull/298/files#r48805334>
> .
>